src/lib/api.js

```diff
@@ -0,0 +1,92 @@
+import {DeployKey} from "./deploy-key.js";
+import {zip} from "zip-a-folder";
+import fs from "fs";
+import {UnknownError} from "./errors.js";
+import parse from "./resource.js";
+import { tmpdir } from "node:os";
+import {sep} from "node:path";
+import {printWhile} from "./cli.js";
+
+export async function apiDeploy(key, deployFolder, releaseNote, {stdin, stdout, stderr}, config) {
+    const project = await DeployKey.getProject(key, { config });
+    const releaseOverviewLink = project.links.get('releases').href;
+    let releaseOverview = await getLinkData(key, releaseOverviewLink, { config });
+    const createFormLink = releaseOverview.links.get('create-form').href;
+    const createFrom = await getLinkData(key, createFormLink, { config });
+    const testDir = fs.mkdtempSync(`${tmpdir()}${sep}`);
```

Comment:
nit: can you import `mkdtempSync` directly instead of `fs` just to be consistent.

Reply:
Done